segar/sim/README.ipynb

```diff
@@ -164,12 +164,12 @@
    "metadata": {},
    "source": [
     "The state contains all attributes necessary to intantiate an object, but a\n",
-    "number of these will be useful for building MDPs (see the `rpp/mdps/README.md`\n",
+    "number of these will be useful for building MDPs (see the `segarar/mdps/README.md`\n",
     "for more details).\n",
     "\n",
     "It's good to be able to see the state of objects, but visualization can be\n",
     "very useful to see what's going on. For this, we borrow a renderer from the\n",
-    "rendering module (see `rpp/rendering/README.md` for details)."
+    "rendering module (see `segarar/rendering/README.md` for details)."
```

Comment:
typo

Reply:
...or maybe it's the new, more immersive version of Segar, Segar-Augmented-Reality or in short SegarAR 😁

Reply:
I don't know what happened in those replaces haha